[CI] Solaris 11x64 rsyslog failed — rsyslog solaris11x64 rsyslog — BUILD FAILED: failed 'gmake -j2 ...' (failure)

## Solaris 11x64 rsyslog CI failure

**Summary:** BUILD FAILED: failed 'gmake -j2 ...' (failure)
**Builder:** rsyslog solaris11x64 rsyslog
**Build:** https://build.rsyslog.com/#/builders/77/builds/7128
**Revision:** bd79ea373259781ec5e67c608f5e5091c5721634
**Worker:** slave-solaris11x64

### Steps

- 0: worker_preparation — success

- 1: set http proxy in git config — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/1/logs/stdio) 

- 2: git — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/2/logs/stdio) 

- 3: shell — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/3/logs/stdio) 

- 4: cleanup — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/4/logs/stdio) 

- 5: cleanup dependencies — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/5/logs/stdio) 

- 6: git branch information — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/6/logs/stdio) 

- 7: building librelp dependency — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/7/logs/stdio) 

- 8: building libfastjson dependency — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/8/logs/stdio) 

- 9: autoreconf for SunStudio Compiler — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/9/logs/stdio) 

- 10: shell_1 — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/10/logs/stdio) [config.log](https://build.rsyslog.com/#/builders/77/builds/7128/steps/10/logs/config_log) 

- 11: build with SunStudio — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/11/logs/stdio) 

- 12: shell_2 — failure — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/12/logs/stdio) [test-suite.log](https://build.rsyslog.com/#/builders/77/builds/7128/steps/12/logs/test-suite_log) 

- 13: gathering make check logs — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/13/logs/stdio) 

- 14: shell_3 — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7128/steps/14/logs/stdio)

--- a/tests/mmexternal-interface-output-none.sh
+++ b/tests/mmexternal-interface-output-none.sh
@@ -2,9 +2,9 @@
 # added 2026-05-22 by Codex, released under ASL 2.0
 # Regression coverage for issue #260: interface.output="none" lets an
 # external message modification helper consume messages without producing the
-# JSON response required by the default protocol. The oracle checks both the
-# downstream omfile action and the helper side-effect file after synchronized
-# shutdown.
+# JSON response required by the default protocol. Because this mode has no
+# helper reply to synchronize on, the oracle waits for the helper side-effect
+# file before shutdown and then checks both that file and downstream omfile.
 . ${srcdir:=.}/diag.sh init
 
 generate_conf
@@ -21,6 +21,7 @@ action(type="omfile" template="outfmt" file="'$RSYSLOG_OUT_LOG'")
 
 startup
 injectmsg literal "<13>Mar 10 01:00:00 host tag:no-output-message"
+wait_file_lines "$RSYSLOG_DYNNAME.side" 1
 shutdown_when_empty
 wait_shutdown
 

